web/src/engine/websites/ThreeHentai.ts

```diff
@@ -0,0 +1,21 @@
+import { Tags } from '../Tags';
+import icon from './ThreeHentai.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { ExtractImageLink } from './NHentai';
+
+@Common.MangaCSS(/^https:\/\/(es\.|fra\.|it\.|pt\.|ru\.)?3hentai\.net\/d\/\d+/, 'div#main-info span.middle-title')
```

Comment:
Maybe use a more generic regex such as `[a-z]{2,3}\.` instead of listing all country codes